src/main/java/it/gov/pagopa/pu/debtpositions/service/statusalign/DebtPositionHierarchyStatusAlignerServiceImpl.java

```diff
@@ -199,4 +203,50 @@ protected DebtPositionDTO alignHierarchyStatusAndRemap(DebtPosition debtPosition
     alignHierarchyStatus(debtPosition);
     return debtPositionMapper.mapToDto(debtPosition);
   }
+
+  private void handleMixedTechDPExpiration(DebtPosition debtPosition, DebtPositionTypeOrg debtPositionTypeOrg, Set<String> iuvs) {
+    if (
+      !DebtPositionOrigin.ORDINARY.equals(debtPosition.getDebtPositionOrigin())
```

Comment:
This is wrong... actually just the origin SPONTANEOUS_SIL could generate them... but it's not a requirement... the only requirement is the dpTypeOrgCode